internal/tracing/init.go

```diff
@@ -39,6 +39,18 @@ better based on experience with this experiment.
 // then we'll enable an experimental OTLP trace exporter.
 const OTELExporterEnvVar = "OTEL_TRACES_EXPORTER"
 
+// OTELTraceParentEnvVar is the env var that should be used to instruct opentofu which
+// trace parent to use.
+// If this environment variable is set when running OpenTofu CLI
+// then we'll extract the traceparent from the environment and add it to the context.
+// This ensures that all opentofu traces are linked to the trace that invoked
+// this command.
+const OTELTraceParentEnvVar = "TRACEPARENT"
+
+// OTELTraceStateEnvVar is the env var that should be used to instruct opentofu which
+// trace state to use.
+const OTELTraceStateEnvVar = "TRACESTATE"
```

Comment:
It seems like these variables are only used as part of the implementation in this package, so should we make them unexported?